src/otx/algo/action_classification/movinet.py

```diff
@@ -25,15 +26,15 @@ class MoViNet(MMActionCompatibleModel):
 
     def __init__(
         self,
-        num_classes: int,
+        label_info: LabelInfoTypes,
```

Comment:
If `label_info` is used to access to some other attributes, it's fine to use it. If it is only passed to access `num_classes`, then it does not make sense to replace `num_classes` with `label_info`. 

`num_classes` is more explicit and clearly shows what it is. As a user, I would not know what `label_info` contains, and need to go to the class definition to look at the details. 

Another comment, maybe not related to this PR,  but `LabelInfoTypes` looks too verbose. Ideally it could be just `LabelInfo`

Hope that helps. Thanks

Reply:
Our expectation for users is to construct the `label_info` object correctly and provide it during model instantiation. This is crucial because we intend for `OTXModel` not to keep anonymity on the model prediction. It's not recommended to simply pass an integer, for example, `OTXModel(num_classes=10)`. This is because of the anonymity I mentioned before. There is no information that the predicted class number is what exactly it is in realty.

However, it should be considered that creating `label_info` from scratch might be cumbersome for users. This object essentially encapsulates dataset characteristics and should originate from the dataset itself. Therefore, the recommended user workflow involves: `data_root -> OTXDataModule -> OTXDataModule.label_info -> OTXModel(label_info)`. I believe there's potential to design a more user-friendly interface to streamline this process further, possibly abbreviating it further to `data_root -> OTXModel(label_info)`.

Reply:
Yeah, I don't disagree with that, and we can think about it more in the future to make it more user-friendly.
Thank you for answers!

Reply:
> However, it should be considered that creating label_info from scratch might be cumbersome for users.

That's my main concern. As a user, I would find it annoying to create another object in addition to datamodule or model. Datamodule should handle this under the hood, and I should not care about creating another object.